group-project/src/main/webapp/index.html

```diff
@@ -1,40 +1,44 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
     <title>SPS - Group 14 Project</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
     <!-- Font Awesome -->
-    <script src="https://kit.fontawesome.com/71959eee97.js" crossorigin="anonymous"></script>
+    <script
+      src="https://kit.fontawesome.com/71959eee97.js"
+      crossorigin="anonymous"
+    ></script>
   </head>
   <body>
     <h1>Google Map Demo</h1>
 
     <div id="input-container">
-        <span><i class="fa-solid fa-street-view fa-xl" id="location-icon"></i></span>
-        <input type="text" placeholder="Enter your location" id="search-input">
+      <span
+        ><i class="fa-solid fa-street-view fa-xl" id="location-icon"></i
+      ></span>
```

Comment:
nit: the alignment and new lines in between the element <span> and <i> tag is kinda weird

Reply:
Wait, is it how Prettier formats the code?